problem_builder/models.py

```diff
@@ -35,6 +35,7 @@ class Answer(models.Model):
     """
 
     class Meta:
+        app_label = 'problem_builder'
```

Comment:
Could you please add a comment here about why this attribute is required?
  Since it's not obvious from the [Django docs](https://docs.djangoproject.com/en/1.8/ref/models/options/#app-label), we want to make sure it isn't deleted down the track.

E.g., something about how it's imported as part of the [`OPTIONAL_APPS`](https://github.com/edx/edx-platform/blob/986afbfa383e2e6bcebfe801c86713f2bae1b9de/lms/envs/common.py#L2795) for lms/Studio, and therefore not added to the `INSTALLED_APPS` list until after it's imported.